DESCRIPTION

```diff
@@ -1,274 +1,274 @@
-    'stat-smooth.r'
-    'stat-sum.r'
-    'stat-summary-2d.r'
-    'stat-summary-bin.R'
-    'stat-summary-hex.r'
-    'stat-summary.r'
-    'stat-unique.r'
-    'stat-ydensity.r'
-    'summary.r'
-    'theme-defaults.r'
-    'theme-elements.r'
-    'theme.r'
-    'translate-qplot-ggplot.r'
-    'translate-qplot-lattice.r'
-    'utilities-break.r'
-    'utilities-grid.r'
-    'utilities-help.r'
-    'utilities-matrix.r'
-    'utilities-resolution.r'
-    'utilities-table.r'
-    'uu_zxx.r'
-    'uu_zzz.r'
-    'z_animint.R'
-    'z_animintHelpers.R'
-    'z_facets.R'
-    'z_geoms.R'
-    'z_gist.R'
-    'z_helperFunctions.R'
-    'z_knitr.R'
-    'z_print.R'
-    'z_scales.R'
-    'z_theme_animint.R'
-    'z_transformShape.R'
-RoxygenNote: 7.1.0
+Package: animint2
+Title: Animated Interactive Grammar of Graphics
+Version: 2020.3.2
+URL: https://github.com/tdhock/animint2
+BugReports: https://github.com/tdhock/animint2/issues
+Authors@R: c(
+    person("Toby", "Hocking",
+     email="toby.hocking@r-project.org",
+     role=c("aut", "cre"),
+     comment="Original animint code"),
+    person("Hadley", "Wickham",
+     role=c("aut"),
+     comment="Forked ggplot2 code"),
+    person("Winston", "Chang",
+     role="aut",
+     comment="Forked ggplot2 code"),
+    person("RStudio",
+     role = c("cph"),
+     comment="Forked ggplot2 code"),
+    person("Nicholas", "Lewin-Koh",
+     role="aut",
+     comment="hexGrob"),
+    person("Martin", "Maechler",
+     role="aut",
+     comment="hexGrob"),
+    person("Randall", "Prium",
+     role="aut",
+     comment="cut_width"),
+    person("Susan", "VanderPlas",
+     role="aut",
+     comment="Animint GSOC 2013"),
+    person("Carson", "Sievert",
+     role="aut",
+     comment="Animint GSOC 2014"),
+    person("Kevin", "Ferris",
+     role="aut",
+     comment="Animint GSOC 2015"),
+    person("Jun", "Cai",
+     role="aut",
+     comment="Animint GSOC 2015"),
+    person("Faizan", "Khan",
+     role="aut",
+     comment="Animint GSOC 2016-2017"),
+    person("Vivek", "Kumar",
+     role="aut",
+     comment="Animint GSOC 2018"))
+Description: Functions are provided for defining animated,
+ interactive data visualizations in R code, and rendering
+ on a web page. The 2018 Journal of Computational and 
+ Graphical Statistics paper,
+ <doi:10.1080/10618600.2018.1513367>
+ describes the concepts implemented.
+Depends:
+    R (>= 3.4.0)
+Imports:
+    digest,
+    RJSONIO,
+    grid,
+    gtable (>= 0.1.1),
+    MASS,
+    plyr (>= 1.7.1),
+    reshape2,
+    scales (>= 0.4.1),
+    stats,
+    tibble,
+    lazyeval
+Suggests:
+    knitr (>= 1.5.33),
+    servr,
+    gistr (>= 0.2),
+    shiny,
+    htmltools,
+    rmarkdown,
+    testthat,
+    RSelenium, XML,
+    devtools,
+    httr,
+    maps,
+    ggplot2movies,
+    hexbin,
+    Hmisc,
+    lattice,
+    mapproj,
+    maptools,
+    mgcv,
+    nlme,
+    quantreg,
+    rpart,
+    svglite
+Enhances: sp
+Remotes: ropensci/RSelenium@v1.7.4

```

Comment:
@tdhock, I've already changed this version thing.

Reply:
in your fork do you have @v1.7.4? i'm confused

Reply:
Yes, and I pushed the same in r4-fixes branch in this repo as well.
https://travis-ci.org/github/lazycipher/animint2/jobs/701227388#L1067